packages/techdocs-common/src/stages/publish/awsS3.test.ts

```diff
@@ -102,6 +102,12 @@ describe('AwsS3Publish', () => {
     });
```

Comment:
Is the file permissions mode change needed? `100755 → 100644`

Reply:
Yep, we want 644 tho, so 👍